packages/terra-consumer-nav/src/components/nav-items/NavItem.scss

```diff
@@ -20,8 +20,7 @@
     text-decoration: none;
 
     &:hover {
-      background: var(--terra-consumer-link-hover-background, rgba(255, 255, 255, 0.4));
-      color: var(--terra-consumer-nav-link-color);
```

Comment:
out of curiosity we don't want 'color' here ?

Reply:
yes, want color. Brought it back.